Tests/test_file_jpeg.py

```diff
@@ -458,7 +458,8 @@ def _n_qtables_helper(n, test_file):
 
             # valid bounds for baseline qtable
             bounds_qtable = [int(s) for s in ("255 1 " * 32).split(None)]
-            self.roundtrip(im, qtables=[bounds_qtable])
+            im2 = self.roundtrip(im, qtables=[bounds_qtable])
+            assert im2.quantization[0] == bounds_qtable
```

Comment:
```suggestion
            assert im2.quantization == {0: bounds_qtable}
```
Just increasing the specificity.